From: SVN-Git Migration <python-modules-team@lists.alioth.debian.org>
Date: Thu, 8 Oct 2015 11:09:41 -0700
Subject: make Sphinx use default theme

Forwarded: not-needed

The "classic" theme is introduced in Sphinx 1.3 and not available in
Sphinx 1.2.

Patch-Name: sphinx-use-default-theme.patch
---
 docs/rftheme/static/rftheme.css_t | 2 +-
 docs/rftheme/theme.conf           | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/rftheme/static/rftheme.css_t b/docs/rftheme/static/rftheme.css_t
index 66aad65..77fe744 100644
--- a/docs/rftheme/static/rftheme.css_t
+++ b/docs/rftheme/static/rftheme.css_t
@@ -1,4 +1,4 @@
-@import url("classic.css");
+@import url("default.css");
 
 .tight-list * {
     line-height: 110% !important;
diff --git a/docs/rftheme/theme.conf b/docs/rftheme/theme.conf
index 1c2b15e..a417128 100644
--- a/docs/rftheme/theme.conf
+++ b/docs/rftheme/theme.conf
@@ -1,4 +1,4 @@
 [theme]
-inherit = classic
+inherit = default
 stylesheet = rftheme.css
 pygments_style = sphinx
